app/src/main/java/com/codesoom/assignment/repository/ProductRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.codesoom.assignment.repository;
+
+import com.codesoom.assignment.entity.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+/**
+ * The interface Product repository.
+ */
```

Comment:
```suggestion
/**
 * 상품 저장소.
 */
```

또는 이렇게 할 수도 있겠죠.

```suggestion
/**
 * 상품 저장소를 추상화합니다.
 */
```

그 외에도 다양한 표현이 가능합니다.